moduleframework/dockerlinter.py

```diff
@@ -244,18 +245,40 @@ def check_clean_all_command(self):
                         return True
         return False
 
-    def check_helpmd_is_present(self):
+    def _get_copy_add_files(self):
         """
-        Function checks if helpmd. is present in COPY or ADD directives
-        :return: True if help.md is present
-                 False if help.md is not specified in Dockerfile
+        Function gets all COPY and ADD files from Dockerfile into list.
+        It contains only source files not target files
+        :return: list
         """
-        helpmd_present = False
+        files = []
         for instruction in [COPY, ADD]:
             try:
-                helpmd = [help for help in self.docker_dict[instruction] if "help.md" in help]
-                if helpmd:
-                    helpmd_present = True
+                # Get only source files, not the target
+                files.extend([x for x in self.docker_dict[instruction] if not x.startswith('/')])
```

Comment:
they can be relative, right? for ADD, those can also be http links

Reply:
I did not know it. Upss.

Reply:
https://docs.docker.com/engine/reference/builder/#add